.travis.yml

```diff
@@ -7,6 +7,9 @@ node_js:
 python:
   - "2.7"
 
+addons:
+  firefox: "latest-nightly"
+
```

Comment:
I'm guessing that we're currently testing against firefox release, meaning that passing our tests suggests that whatever code we've got is likely to be upliftable to release.   I'm also guessing that this switches us from release to nightly, meaning we lose early feedback about landing stuff that's not upliftable, but we gain confidence that this will work right on nightly.  Is that correct?

Assuming it is, it's not obvious to me which of these choices is the right tradeoff (in some ideal world, we'd test against both, and maybe beta too, but that's probably for another day...).

What motivated this change here?

Whatever we choose, I think we want a comment explaining the tradeoffs and the choice made in the yml file.
That might be the right tradefo

Reply:
Travis uses Firefox 56 by default which is missing some Javascript API features (we've hit this before like in [this comment](https://github.com/mozilla/activity-stream/blob/ac76f59c25e8561030217df099636887a39cd690/test/unit/asrouter/CFRPageActions.test.js#L117-L118)).
While removing some of the babel plugins I ran into the same issue.
I thought to set this to `latest-nightly` because that's where our code will land.

Reply:
Just to clarify: `latest-nightly` is not needed, it could just as well be `stable`, it was just the fact that we land in Nightly.

Reply:
@k88hudson do you have an opinion on what version we should use here?  I'm not actually sure how much it matters what we choose here, are you aware of us having hit any problems that would have been caught earlier had we been using a newer version on travis?  If not, maybe this isn't worth overthinking.

Reply:
Personally I think it makes sense to use `latest-nightly` since we want to be in line with what firefox tests would run against (and presumably if we run tests in tree, that's what they would run against)

Reply:
That sounds reasonable to me.  @piatra let's go with that, and add a comment explaining the reasoning please.